pkgs/games/gnujump/default.nix

```diff
@@ -11,6 +20,11 @@ stdenv.mkDerivation rec {
 
   NIX_LDFLAGS = "-lm";
 
+  postInstall = ''
+    install -Dm644 -t $out/share/applications ${desktopItem}/share/applications/*
```

Comment:
Use `copyDesktopItems`

Reply:
Oh, awesome, thank you for the tip! That's a nice handy tool. Fixed.